setup.cfg

```diff
@@ -0,0 +1,27 @@
+[metadata]
```

Comment:
What do you think about porting this to a `pyproject.toml` instead and lean fully into current Python packaging standards? Would go well with our plans around conda plugins via pyproject.toml.

Reply:
All for it. I didn't understand why _two_ files were needed, but the PyPA example and others I've seen live had all of it 🤷